docs/CONFIG.md

```diff
@@ -9243,6 +9308,33 @@ TipCapMinimum is the minimum gas tip to use when submitting transactions to the
 
 (Only applies to EIP-1559 transactions)
 
+## EVM.GasEstimator.DAOracle
+```toml
+[EVM.GasEstimator.DAOracle]
+OracleType = 'opstack' # Example
+OracleAddress = '0x420000000000000000000000000000000000000F' # Example
+CustomGasPriceCalldata = '' # Default
+```
+
+
+### OracleType
+```toml
+OracleType = 'opstack' # Example
+```
+OracleType refers to the oracle family this config belongs to. Currently the available oracle types are: 'opstack', 'arbitrum', and 'zksync'.
+
+### OracleAddress
+```toml
+OracleAddress = '0x420000000000000000000000000000000000000F' # Example
+```
+OracleAddress is the address of the oracle contract.
+
+### CustomGasPriceCalldata
+```toml
+CustomGasPriceCalldata = '' # Default
+```
+CustomGasPriceCalldata is optional and can be set to call a custom gas price API at the given OracleAddress.
```

Comment:
/nit Should the field be call CustomGasPriceAPI?

Reply:
haha we already had a debate offline and the consensus seemed to be that API was too much of a web2 concept and was odd to reference here. I'll update this documentation though to not say "API"